File: cash.c
#include <cs50.h>
#include <stdio.h>
#include <math.h>

int main(void)
{
    float change; //Declare the variable
    int coins = 0; //Initializes coins to 0
    do
    {
        change = get_float("Change owed:"); //Get the height
    }
    while (!(change >= 0)); //Re ask for the height if value < 0
    int cents = round(change * 100);
    coins += round(cents / 25);
    cents -= round(cents / 25) * 25;
    coins += round(cents / 10);
    cents -= round(cents / 10) * 10;
    coins += round(cents / 5);
    cents -= round(cents / 5) * 5;
    coins += round(cents);
    printf("%d", coins);
}

File: cash.py
from cs50 import get_float


# Pilot code


def main():
    # Gets change from user
    change = get_positive_float()*100
    # Gets the number of quarters
    quarters = change // 25
    # Update change
    change -= quarters * 25
    # Gets the number of dimes
    dimes = change // 10
    # Update change
    change -= dimes * 10
    # Gets the number of nickels
    nickels = change // 5
    # Update change
    change -= nickels * 5
    # Gets the number of pennies
    pennies = change
    print(round(quarters+dimes+nickels+pennies))


# Sanitizes the inputs to get a positive integer


def get_positive_float():
    while True:
        n = get_float("Change owed: ")
        if n > 0:
            break
    return n


main()
